src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -38,6 +38,34 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if any word in the {@code sentence} starts the {@code word}.
+     *   Ignores case, and a substring match suffices.
+     *   <br>examples:<pre>
+     *       containsStartSubstringIgnoreCase("ABc def", "ab") == true
+     *       containsStartSubstringIgnoreCase("ABc def", "de") == true
+     *       containsStartSubstringIgnoreCase("ABc def", "abc d") == true
+     *       containsStartSubstringIgnoreCase("ABc def", "ef") == false //not the starting part of the word.
+     *       </pre>
+     * @param sentence cannot be null
+     * @param word cannot be null, cannot be empty, must be a single word
```

Comment:
i think word does not have to be a single word now, it could have multiple words

Reply:
Should be resolved now, thank you for the feedback!